docs/core-concepts/inter-step-buffer-service.md

```diff
@@ -73,6 +73,47 @@ spec:
       volumeSize: 10Gi # Optional, defaults to 20Gi
 ```
 
+### PDB
+
+PDB (Pod Disruption Budget) is essential for running ISB in production to ensure availability.
+
+#### Example ISB with PDB
+
+```yaml
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: InterStepBufferService
+metadata:
+  name: default
+spec:
+  jetstream:
+    version: latest
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchLabels:
+                  app.kubernetes.io/component: isbsvc
+                  numaflow.numaproj.io/isbsvc-name: default
+              topologyKey: topology.kubernetes.io/zone
+            weight: 100
```

Comment:
Could you add the persistence and replicas here?
Just to help anyone copying from here

Reply:
Actually, this section can be split (or together for availability and resiliency) for anti-affinity and PDB

Reply:
Can give some context on  anti-affinity

Reply:
@kohlisid how about https://github.com/numaproj/numaflow/pull/2787/commits/b5fff04423b6ad4f10ac109550a30a2d48511b5e? I don't want to add more link, they can figure those out. WDYT?